### PR TITLE
RET-4192: Restore old caseFields

### PR DESCRIPTION
**Pull request by fergusoshea**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/RET-4192

### Change description ###
Ensure fields for old flow of InitialConsideration continue to exist

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by lvo700 — APPROVED** [review]

File: definitions/json/CaseField/CaseField.json [hunk 1]
```diff
@@ -1287,7 +1287,7 @@
   },
   {
     "CaseTypeID": "ET_EnglandWales",
-    "ID": "etInitialConsiderationBeforeYouStart",
+    "ID": "initialConsiderationBeforeYouStart",
```
Comment (hsjhita) [inline definitions/json/CaseField/CaseField.json:1290]: this also needs to be on CaseEventToFields - https://github.com/hmcts/et-ccd-definitions-englandwales/blob/ffc1472ef35a327427337d557b833770b0f42495/definitions/json/CaseEventToFields/CaseEventToFields.json#L3538C38-L3538C38

File: definitions/json/AuthorisationCaseField/AuthorisationCaseField.json [hunk 2]
```diff
@@ -12601,40 +12601,400 @@
   },
   {
     "CaseTypeId": "ET_EnglandWales",
-    "CaseFieldID": "etInitialConsiderationBeforeYouStart",
+    "CaseFieldID": "initialConsiderationBeforeYouStart",
```
Comment (hsjhita) [inline definitions/json/AuthorisationCaseField/AuthorisationCaseField.json:12604]: legalrepsolicitor shouldn't have access to IC event or fields

Comment (hsjhita) [reply definitions/json/AuthorisationCaseField/AuthorisationCaseField.json:12604]: even if it is D

File: definitions/json/CaseTypeTab/CaseTypeTab.json [hunk 3]
```diff
@@ -928,160 +928,229 @@
...
+    "CaseFieldID": "icCompletedBy",
+    "TabFieldDisplayOrder": 12
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "Channel": "CaseWorker",
+    "TabID": "initialConsideration",
+    "CaseFieldID": "icDateCompleted",
+    "TabFieldDisplayOrder": 13
+  },
```
Comment (hsjhita) [inline definitions/json/CaseTypeTab/CaseTypeTab.json:1026]: these should be the last 2 fields on the tab

**Review by hsjhita — CHANGES_REQUESTED** [review]

File: definitions/json/CaseField/CaseField.json (hunk 1 above)
Comment (fergusoshea) [reply definitions/json/CaseField/CaseField.json:1290]: Done

File: definitions/json/CaseTypeTab/CaseTypeTab.json (hunk 3 above)
Comment (fergusoshea) [reply definitions/json/CaseTypeTab/CaseTypeTab.json:1026]: I have put etICJuridictionCodesInvalid & etICInvalidDetails to the bottom of the list

File: definitions/json/AuthorisationCaseField/AuthorisationCaseField.json (hunk 2 above)
Comment (fergusoshea) [reply definitions/json/AuthorisationCaseField/AuthorisationCaseField.json:12604]: Removed

File: definitions/json/CaseTypeTab/CaseTypeTab.json (hunk 3 above)
Comment (hsjhita) [reply definitions/json/CaseTypeTab/CaseTypeTab.json:1026]: the date and who it was completed by should be the last 2 things on the tab

Comment (fergusoshea) [reply definitions/json/CaseTypeTab/CaseTypeTab.json]: Ah sorry, I picked the wrong two. I have moved data and who to the bottom of the list

**Review by hsjhita — APPROVED** [review]

**Pull request merged**